db/migrate/20151102113518_add_default_value_to_minimum_fixed_fee.rb

```diff
@@ -0,0 +1,12 @@
+class AddDefaultValueToMinimumFixedFee < ActiveRecord::Migration
+  def up
+    firms = Firm.where(minimum_fixed_fee: nil)
+    firms.each{|f| f.update_attribute('minimum_fixed_fee', 0)}
```

Comment:
I know we don't have Rubocop on this repo yet, but formatting here needs spaces `each { |f| .... 0) }`